src/Package/Impl/Help/HelpVisualComponent.cs

```diff
@@ -266,6 +266,9 @@ internal sealed class HelpVisualComponent : IHelpVisualComponent {
 
         private static bool IsHelpUrl(string url) {
             Uri uri = new Uri(url);
+            if(uri.AbsoluteUri.EndsWithIgnoreCase(".pdf")) {
```

Comment:
That would difficult to solve since we don't know content then. R Vignettes seem to work normally (i.e. they do have pdf extension)